standard/lexical-structure.md

```diff
@@ -909,10 +947,15 @@ Pre-processing directives are not tokens and are not part of the syntactic gramm
 The conditional compilation functionality provided by the `#if`, `#elif`, `#else`, and `#endif` directives is controlled through pre-processing expressions ([§7.5.3](lexical-structure.md#753-pre-processing-expressions)) and conditional compilation symbols.
 
 ```ANTLR
-Conditional_Symbol
-    : Identifier_Or_Keyword { IsNotTrueOrFalse() }?
+fragment Conditional_Symbol
+    : Basic_Identifier           // excluding TRUE and FALSE
+         { IsNotTrueOrFalse() }? // see note below
     ;
 ```
+
+> *Note* The ANTLR semantic predicate above: `IsNotTrueOrFalse`; is informative *only*. How a compiler enforces the restriction on the allowable
```

Comment:
As before, I'd reword this a bit:

"The ANTLR semantic predicate above (`IsNotTrueOrFalse`) is informative *only*."

Reply:
Changed